packages/ocaml-config/ocaml-config.2/opam

```diff
@@ -12,10 +12,12 @@ bug-reports: "https://github.com/ocaml/opam/issues"
 depends: [
   "ocaml-base-compiler" {>= "4.12.0~"} |
   "ocaml-variants" {>= "4.12.0~"} |
-  "ocaml-system" {>= "4.12.0~"}
+  "ocaml-system" {>= "4.12.0~"} |
```

Comment:
We definitely can't edit the `ocaml-config.2` package, because that will trigger mass rebuilds for existing switches - it's OK to add a new ocaml-config.3 package and the Windows fixes will be very welcome in the main script. For ocaml-config.3, the constraints above can all be `>= "5.0.0~"`

Reply:
Okay I made https://github.com/ocaml/opam-repository/pull/21306 for ocaml-config.3